src/hotspot/cpu/riscv/sharedRuntime_riscv.cpp

```diff
@@ -610,7 +610,7 @@ AdapterHandlerEntry* SharedRuntime::generate_i2c2i_adapters(MacroAssembler *masm
   Label skip_fixup;
 
   const Register receiver = j_rarg0;
-  const Register data = t1;
+  const Register data = t0;
   const Register tmp = t2;  // A call-clobbered register not used for arg passing
```

Comment:
Can you remove following line at L614 while you are on it? Seems a left-over which is not used anywhere.

```
const Register tmp = t2;  // A call-clobbered register not used for arg passing
```

Reply:
Yes